ksa_compliance/output_models/e_invoice_output_model.py

```diff
@@ -248,15 +248,16 @@ def __init__(self,
                                 parent="invoice")
 
         if self.additional_fields_doc.allowance_indicator:
-            self.get_float_value(field_name="sum_of_allowances",
```

Comment:
If we're no longer using sum_of_allowances, we should remove it from sales invoice additional fields